packages/dashboard-hardhat-plugin/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@truffle/dashboard-hardhat-plugin",
+  "version": "0.1.0-0",
+  "description": "Truffle Dashboard for Hardhat plugin",
+  "repository": "github:trufflesuite/dashboard-hardhat-plugin",
+  "author": "ConsenSys Software Inc",
+  "license": "MIT",
+  "main": "dist/src/index.js",
+  "types": "dist/src/index.d.ts",
+  "keywords": [
+    "ethereum",
+    "smart-contracts",
+    "hardhat",
+    "hardhat-plugin",
+    "truffle",
+    "truffle-dashboard"
+  ],
+  "scripts": {
+    "lint:fix": "prettier --write 'src/**/*.{js,ts}' 'test/**/*.{js,ts}' && tslint --fix --config tslint.json --project tsconfig.json",
+    "test": "mocha --timeout 10000",
+    "build": "tsc",
+    "watch": "tsc -w",
+    "prepublishOnly": "npm run build"
```

Comment:
```suggestion
```

Probably don't need this, since `prepare` will suffice.

Reply:
Same as above (_removed in a separate commit..._) 👍